classes/access_manager.php

```diff
@@ -132,11 +129,16 @@ private function validate_config_key(string $pageurl = '') : bool {
 
     /**
      * Check if Safe Exam Browser is required to access quiz.
+     * If quizsettings do not exist, then there is no requirement for using SEB.
      *
      * @return bool If required.
      */
     public function seb_required() : bool {
-        return $this->quizsettings->get('requiresafeexambrowser') != settings_provider::USE_SEB_NO;
+        if (!$this->quizsettings) {
```

Comment:
I can't see that we cover this with tests. Can we cover this please? Or point me where we do it.